src/WebJobs.Extensions/Extensions/Files/Bindings/FileTriggerAttributeBindingProvider.cs

```diff
@@ -17,12 +17,12 @@ namespace Microsoft.Azure.WebJobs.Extensions.Files.Bindings
 {
     internal class FileTriggerAttributeBindingProvider : ITriggerBindingProvider
     {
-        private readonly FilesConfiguration _config;
+        private readonly FilesOptions _options;
         private readonly ILogger _logger;
 
-        public FileTriggerAttributeBindingProvider(FilesConfiguration config, ILoggerFactory loggerFactory)
+        public FileTriggerAttributeBindingProvider(FilesOptions options, ILoggerFactory loggerFactory)
         {
-            _config = config ?? throw new ArgumentNullException("config");
```

Comment:
nit: mismatched parameter name. Update to use nameof

Reply:
Fixed